11-spatial-cv.Rmd

```diff
@@ -43,15 +43,15 @@ By contrast, statistical inference is impossible with machine learning [@james_i
 <!-- The primary aim of machine learning is to make good predictions, whereas the field of statistics is more focussed on the underlying theory [e.g. @zuur_mixed_2009] -->
 
 The primary aim of much machine learning research is to make good predictions, as opposed to statistical/Bayesian inference, which is good at helping to understand underlying mechanisms and uncertainties in the data [see @krainski_advanced_2018].
-Machine learning thrives in the age of 'big data' because its methods makes few assumptions about input variables and can handle huge datasets.
+Machine learning thrives in the age of 'big data' because its methods make few assumptions about input variables and can handle huge datasets.
 Machine learning is conducive to tasks such as the prediction of future customer behavior, recommendation services (music, movies, what to buy next), face recognition, autonomous driving, text classification and predictive maintenance (infrastructure, industry).
 
 <!-- ^[In this case we do not have too worry too much about possible model misspecifications since we explicitly do not want to do statistical inference.] -->
 
 This chapter is based on a case study: the (spatial) prediction of landslides.
 This application links to the applied nature of geocomputation, defined in Chapter \@ref(intro), and illustrates how machine learning borrows from the field of statistics when the sole aim is prediction.
 Therefore, this chapter first introduces modeling and cross-validation concepts with the help of a Generalized Linear Model [GLM; @zuur_mixed_2009].
-Building on this the chapter implements a more typical machine learning algorithm, namely a Support Vector Machine (SVM).
+Building on this, the chapter implements a more typical machine learning algorithm, namely a Support Vector Machine (SVM).
```

Comment:
Another example of the 'comma-conservative' style of the previous book. Seems that 'comma-happy' is the way to be :smile: